x-pack/plugins/lists/common/schemas/request/create_exception_list_item_schema.ts

```diff
@@ -25,8 +25,8 @@ import { RequiredKeepUndefined } from '../../types';
 import {
   CreateCommentsArray,
   DefaultCreateCommentsArray,
-  DefaultEntryArray,
   NamespaceType,
+  nonEmptyEntriesArray,
```

Comment:
Nit: these seem to normally be exported as uppercase constants. I know you're exporting `TypeOf` with that name, but maybe we need a convention for that if there isn't one.

Reply:
Yea, spoke with @FrankHassanabad about that earlier today. He'd suggested going lowercase here to differentiate as throughout we tend to use the lowercase for io-ts. Def something we should discuss to settle on one way or the other.